pom.xml

```diff
@@ -270,7 +270,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-failsafe-plugin</artifactId>
-                <version>2.22.1</version>
+                <version>3.0.0-M3</version>
```

Comment:
Looking into notes here https://maven.apache.org/surefire/maven-failsafe-plugin/ I think it's fine to upgrade, this will also prepare us for Java 11 upgrade, so it is better to know any possible problems in advance